semantics/language/translation/env.k

```diff
@@ -105,4 +106,32 @@ module C-ENV
                ListItem(L:LocalCell) => .List
           ...</block-stack>
 
+     rule recover ~> (K:KItem => .K)
+          requires K =/=K popLocals
+     rule (recover => .K) ~> popLocals
```

Comment:
one thing I'm not entirely clear on here though is, if the conditional expression contains an error, how does "recover" eventually lead it back to having a type?